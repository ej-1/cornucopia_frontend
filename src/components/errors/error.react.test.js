import React from "react";
import Error from "../errors/error";
import renderer from "react-test-renderer";
import { shallow } from "enzyme";

test("Error gets rendered", () => {
  const component = renderer.create(<Error message={"Danger! Danger!"} />);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});

test("Error without message gets rendered", () => {
  const wrapper = shallow(<Error />);
  expect(wrapper.find("div").html()).toEqual(
    '<div class="error">Something went wrong.</div>'
  );
});
