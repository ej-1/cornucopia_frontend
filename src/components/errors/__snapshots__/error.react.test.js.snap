// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Error gets rendered 1`] = `
<div
  className="error"
>
  Danger! Danger!
</div>
`;
